### ISSUE TITLE
Categorization Analysis

### ISSUE THREAD

**Issue by mattskier** [mattskier/ravens-dse-511#14]
I wrote basic codes, any ideas for extending that much appreciated, accuracy rate ~76% using idealized hyperparameters

**Comment by mattskier** [comment]
-will need to work through SVM analysis as well today, completed a decision tree analysis available in repository

**Comment by mattskier** [comment]
completed SVM/Decision tree classifiers and necessary cross validation, updated in categorical.ipynb notebook, will work on report later today

**Comment by mattskier** [comment]
completed discussion for decision tree/SVM on google doc, will let Prachi discuss random forest and Suann to discuss anything else devaloped

**Comment by prachi1294** [comment]
Yes, I have discussed RF and updated the notebook DM_reduction_data.ipynb
